interfaces/builtin/steam_support.go

```diff
@@ -0,0 +1,201 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016-2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const steamSupportSummary = `allows access to various permissions required to use Steam`
+
+const steamSupportBaseDeclarationSlots = `
+  steam-support:
+    deny-connection: true
+    deny-auto-connection: true
+`
+
```

Comment:
Please add a comment here:
```
// Steam games sometimes use the chromium content API where we would
// normally use the browser-support interface. Other steam games require
// ptracing their children but the browser-support interface explicitly denies
// ptrace since it can be used to break out of the seccomp sandbox on <4.8
// kernels. As such, grab the rules from browser-support that we can use
// here and add the extra syscalls we need.
```